tracer/src/Datadog.Trace/Ci/Agent/ApmAgentWriter.cs

```diff
@@ -15,17 +14,17 @@
 namespace Datadog.Trace.Ci.Agent
 {
     /// <summary>
-    /// CI Visibility Agent Writer
+    /// APM Agent Writer
```

Comment:
Try to avoid confusion with the tracer's `AgentWriter` :) 
```suggestion
    /// APM Agent Writer for CI Visibility
```